bin/console

```diff
@@ -4,13 +4,25 @@
 declare(strict_types=1);
 
 use Antidot\Cli\Application\Console;
+use Antidot\Framework\Runtime\AntidotRuntime;
+
+ini_set('memory_limit', '2048M');
+
+$_SERVER['APP_RUNTIME'] = AntidotRuntime::class;
+$_SERVER['APP_RUNTIME_OPTIONS'] = [
+    'host' => '0.0.0.0',
+    'port' => 3000,
```

Comment:
I think we should let `3000` port number be used before running the development server.

Or we can have the setting to set these application runtime options.

Reply:
It should be configurable from the config 100%

Reply:
Done, now the server is configured as every other component at `config/services(dependencies.prod.yaml`